src/views/Home.jsx

```diff
@@ -1,22 +1,72 @@
+import { useEffect } from 'react';
+import { useState } from 'react';
 import { generateToken } from '@the-collab-lab/shopping-list-utils';
-import { useNavigate } from 'react-router-dom';
+import { streamListItems } from '../api';
 import './Home.css';
 
 export function Home({ handleListTokenState }) {
-	const navigate = useNavigate();
+	const [userEnteredToken, setUserEnteredToken] = useState('');
+	const [message, setMessage] = useState('');
+
+	useEffect(() => {
+		const timer = setTimeout(() => {
```

Comment:
I like that the error message timeout is consistent across the app now.  I think we should try to remember this going forward for other user issues as well.